app/assets/stylesheets/blacklight/_facets.scss

```diff
@@ -70,27 +68,22 @@
   }
 
   .facet-label {
-    display: table-cell;
     padding-right: 1em;
     text-indent: -15px;
     padding-left: 15px;
-    padding-bottom: $padding-base-vertical;
     @include hyphens-auto;
   }
 
   .facet-count {
-    display: table-cell;
```

Comment:
There's a lot of history behind the facet count displays. My memory is `display: table-cell` is extremely important to getting good text wrapping to happen when there's long labels and long counts. Let me see if I can come up with a good example.

Reply:
@cbeer OK! My bad. Thanks.

Reply:
I am seeing something funny with the first value of each facet (but this isn't what i was talking about):

![Screen Shot 2022-07-20 at 13 46 35](https://user-images.githubusercontent.com/111218/180078572-47ce7a72-b939-478c-b947-696a37563bd5.png)

Reply:
@cbeer  I think I see the wrapping issue too. I'll take more time on it, or can abandon if you think it's too deeply enmeshed.

Reply:
The kinds of facet values I was concerned about seem just fine:
![Screen Shot 2022-07-20 at 13 51 50](https://user-images.githubusercontent.com/111218/180079812-6509de17-608d-4efe-bd55-53ed4bfa0bdd.png)

Flexbox for the win? It might be good to port this upstream (https://github.com/projectblacklight/blacklight/blob/main/app/assets/stylesheets/blacklight/_facets.scss) and see if anyone else remembers what else we're worried about.